crates/ln-dlc-node/src/ln/coordinator_event_handler.rs

```diff
@@ -366,7 +368,7 @@ where
             }
 
             tracing::debug!(
-                %target_node_id,
+                %trader_id,
```

Comment:
:wrench: Prefer consistent naming i.e. `peer_id`.

Reply:
ok, changed it.